resources/resources.go

```diff
@@ -0,0 +1,194 @@
+package resources
```

Comment:
We cannot understand what this package does from this package name. (`resouces` is too generic a name.)
I think `package export` is better...? I'm not sure, because this service also does import. 😅